docs/source/endpoints/navroot.md

```diff
@@ -0,0 +1,159 @@
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_site_get.req
+```
+
+The response will contain the navigation root information of the root of the site:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_site_get.resp
+:language: http
+```
+
+In a multilingual site where the language folder is the navigation root, the response has the language folder information:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_lang_folder_get.req
+```
+
+The response will contain the navigation root information for the site:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_lang_folder_get.resp
+:language: http
+```
+
+In a multilingual site, if the navigation root is requested for content inside a language folder:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_lang_content_get.req
+```
+
+The response has the language folder information as a navigation root:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_lang_content_get.resp
+:language: http
+```
+
+(navigation-root-expansion-label)=
+
+## Expansion
+
+This endpoint can be used with the {doc}`expansion` mechanism which allows getting more information about a content item in one query, avoiding unnecessary requests.
+
+If a simple `GET` request is done on the content item, a new entry will be shown on the `@components` entry with the URL of the `@navroot` endpoint.
+
+In a standard site when querying the site root:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_get_expansion.req
+```
+
+The response will contain information of the site root with the navigation expanded:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_get_expansion.resp
+:language: http
+```
+
+When querying a content type inside the root:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_content_get_expansion.req
+```
+
+The response will contain the information of that content and navigation root information with the site expanded:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_content_get_expansion.resp
+:language: http
+```
+
+In a multilingual site, it will work similarly. This will be the request:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/site_get_expand_navroot.req
+```
+
+And the response will contain the navigation root information pointing to the root of the site:
+
+```{literalinclude} ../../src/plone/restapi/tests/http-examples/site_get_expand_navroot.resp
```

Comment:
Check the path. This literalinclude file does not show up in the rendered preview.

![Screen Shot 2023-04-12 at 4 34 40 AM](https://user-images.githubusercontent.com/102112/231445137-2436fc5e-7484-4e93-a64c-f4c48467e03d.png)

Reply:
I see it now. Yay!